### DISCUSSION TITLE
Log output without newline, and can't interrupt sshuttle (ctrl+c)

### DISCUSSION THREAD

**Discussion by lucianomx** [sshuttle/sshuttle#757] [General]
I have just installed sshuttle into Ubuntu 22.04, its log output is a mess without newline and I **can't interrupt it with CTRL+C**. Is there any trick to fix that? At least the CTRL+C support?
(shuttle installed: 1.0.5, by: `apt install sshutle`)

Output example:
![image](https://user-images.githubusercontent.com/1851108/167635000-8e44189f-2e85-4d8c-85da-3e6b5837dfd6.png)

**Comment by tristanc** [comment]
This is happening to me too after I upgraded to more recent version of Ubuntu.  It used to work correctly before.

**Reply by qadzek** [reply]
Try the solution described in this [issue](https://github.com/sshuttle/sshuttle/issues/804).
